env/CMakeLists.txt

```diff
@@ -45,6 +45,7 @@ ExternalProject_Add(
     -DPython3_EXECUTABLE=${TTMLIR_TOOLCHAIN_DIR}/venv/bin/python
     -DCMAKE_INSTALL_PREFIX=${TTMLIR_TOOLCHAIN_DIR}
     -DLLVM_ENABLE_PROJECTS=mlir
+    -DLLVM_ENABLE_RTTI=ON
```

Comment:
Ooof, this sucks :(

Ok this is actually in deal breaker territory, let's meet next week to discuss our options.  I'll also file an issue on onnx-mlir to see if they ever plan on removing this requirement: https://github.com/onnx/onnx-mlir/discussions/2897